src/model/utilities/utilities.test.js

```diff
@@ -0,0 +1,55 @@
+import { jsxEmptyExpression } from '@babel/types';
+import {randomNumbers, randomValueFromArray} from './utilities'
+
+describe('randomNumbers', ()=>{
+  let intro = 'If we set lowest number as 1, highest number as 4 and length as 4 then we should';
+  test(`${intro} get array with numbers 1, 2, 3, 4`, ()=> {
+    const result = randomNumbers(1, 4, 4);
+    expect(result).toContain(1, 2, 3, 4);
+  });
+
+  test(`${intro}n't get array with numbers 0, 5`, ()=> {
+    const result = randomNumbers(1, 4, 4);
+    expect(result).not.toContain(5);
+  });
```

Comment:
```suggestion
describe('randomNumbers', ()=>{
  describe('If we set lowest number as 1, highest number as 4 and length as 4', ()=>{
    const result = randomNumbers(1, 4, 4);
    test(`then we should get array with numbers 1, 2, 3, 4`, ()=> {
       expect(result).toContain(1, 2, 3, 4);
     });
  
     test(`then we should get array with numbers 0, 5`, ()=> {
        expect(result).not.toContain(5);
      });
```

Reply:
I tak dalej :)

Reply:
Nice! W ogóle nie pomyślałem, żeby to tak ładnie skrócić. Dzięki za rade! I jak coś, to zrobione :D